src/greenlight/step.clj

```diff
@@ -272,6 +272,15 @@
     (fn? (::title step)) (update ::title #(% ctx))))
 
 
+(defmulti report->outcome :type)
+
+(defmethod report->outcome :pass [_] :greenlight.assert/pass)
```

Comment:
Seems like we might want to make `greenlight.assert` a proper namespace if we expect additional customization. Some of the reporting and failure detection code below in `advance!` could also move there.